DESIGN.md

```diff
@@ -73,77 +157,225 @@ Flow control support:
 - buffering, sampling, throttling, windowing, dropping, etc
 - temporal and count-based strategies
 - `request(n)` consumer demand signal
-  - for pull-based sources, this allows batched "async pull"
-  - for push-based sources, this allows backpressure signals to conditionally apply strategies (i.e. drop, buffer, sample, fail, etc)
+	- for pull-based sources, this allows batched "async pull"
+	- for push-based sources, this allows backpressure signals to conditionally apply strategies (i.e. drop, first, buffer, sample, fail, etc)
+
+You get a flowable from:
+
+- Converting a Observable with a backpressure strategy
+- Create from sync/async OnSubscribe API (which participate in backpressure semantics)
+
+*Type Signature*
+
+```java
+class Flowable<T> implements Flow.Publisher<T>, io.reactivestreams.Publisher<T> {
+  void subscribe(Subscriber<T> subscriber);
+
+  interface Subscriber<T> implements Flow.Subscriber<T>, io.reactivestreams.Subscriber<T> {
```

Comment:
Right now we are not targeting `Flow.Subscriber` since this planned for Java 1.9. Our current source code compatibility is 1.6. I believe the rationale was to show that we intend that this would one day extend and interoperate with `Flow`. 

This line should just be changed to ...

```java
interface Subscriber<T> extends io.reactivestreams.Subscriber<T> {
```